src/util/std_expr.h

```diff
@@ -1012,8 +955,7 @@ inline void validate_expr(const minus_exprt &value)
 }
 
 
-/*! \brief binary multiplication
-*/
+/// \brief binary multiplication
```

Comment:
Associativity?

Reply:
Same as for sum: I don't know where to get the information.

Reply:
Also for me: same as sum: https://github.com/diffblue/cbmc/pull/2800#discussion_r211978321 :-)

Reply:
Nit: uppercase B